pkg/external/external.go

```diff
@@ -211,6 +211,10 @@ type SlimCred struct {
 	CapAmbient     uint64 /* Ambient capability set */
 }
 
+func (s SlimCred) GetSizeBytes() uint32 {
+	return 80
+}
+
```

Comment:
you mean instead of having `externals.SlimCred` object we will have `bufferdecoder.SlimCred` (under file protocol.go)? if so, then totally fine to me.

Reply:
Not instead, in addition. We will still need SlimCred to be exposed in external so tracee-rules can use it

Reply:
but then you will have code duplication and and it will make things prone to error, as every update on `externals.SlimCred` will require also an update to `bufferdecoder.SlimCred` (which people might forget to do so).
I can be kind of OK with code dup, but the coupling between these two structures bothers me. They are exactly the same but defined twice.

Reply:
alright, you have a point. done :)